packages/manager/core/generator/app/conditional-templates/listing/listing.test.ts.hbs

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from '@playwright/test';
+import { login, logout } from '../../../../../playwright/login';
+import * as config from '../../../../../playwright/config.json';
+
+test('Listing page test', async ({ page }) => {
+  await page.goto(config.appUrl);
+  await login(page);
+  await page.waitForSelector('h2');
+  const h2Text = await page.textContent('h2');
+  expect(h2Text).toContain('Services list');
```

Comment:
same remark as [packages/manager/core/generator/app/conditional-templates/default/home.test.ts.hbs](https://github.com/ovh/manager/pull/9365/files#diff-cdce3280f40d1da5bb6a2a5198f6dec8129b9c4b83a9f06c27dc4a9bd4049519)
you shouldn't need "waitForSelector"

Reply:
Done ✅